app/helpers/service_provider_helper.rb

```diff
@@ -73,9 +73,9 @@ def yamlized_sp(service_provider)
     yamlable_json.to_yaml.delete('\"')
   end
 
-  def sp_active_img_alt(service_provider_is_active)
-    return 'Active service provider' if service_provider_is_active
-    'Inactive service provider'
+  def sp_active_img_alt(service_provider_is_accessible)
```

Comment:
hm is this method in use? i only see it referenced in tests. i think we can axe this, since we add the alt text directly to the image.